resources/views/store/products/supporter-tag.blade.php

```diff
@@ -0,0 +1,64 @@
+{{--
+    Copyright 2015-2017 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+@if(!Auth::user())
+<div class="grid grid--gutters">
+    <div class="grid-cell grid-cell--1of2">
+        @php
+            $title = __('users.anonymous.login_link');
+            $text = __('store.supporter_tag.require_login.link_text');
+            $link = Html::link('#', $text, ['class' => 'js-user-link', 'title' => $title]);
+        @endphp
+        {!! __('store.supporter_tag.require_login._', ['link' => $link]) !!}
+    </div>
+</div>
+@else
+<div class="js-store-supporter-tag store-supporter-tag"
+    data-username="{{ Auth::user()->username }}" data-avatar-url="{{ Auth::user()->user_avatar }}">
+    <input type="hidden" name="item[product_id]" value="{{ $product->product_id }}" />
+    <input type="hidden" name="item[quantity]" class="js-store-item-quantity" value="1" />
+    <input type="hidden" id="supporter-tag-form-price" name="item[cost]" value="4" />
+    <input type="hidden" name="item[extra_data][duration]" value="1" />
```

Comment:
I think this should be calculated and assigned server-side as well, since it would help make the payment callback validation simpler